src/Workspaces/CSharpTest/CodeGeneration/SyntaxGeneratorTests.cs

```diff
@@ -159,8 +159,8 @@ public class MyAttribute : Attribute { public MyAttribute(E value) { } }",
             VerifySyntax<AttributeListSyntax>(_g.Attribute(GetAttributeData(
 @"using System; 
 public class MyAttribute : Attribute { public MyAttribute(Type value) { } }",
-@"[MyAttribute(typeof(MyAttribute))]")),
-@"[global::MyAttribute(typeof (global::MyAttribute))]");
+@"[MyAttribute(typeof (MyAttribute))]")),
```

Comment:
Isn't this a regression? Diff seems to have changed from:
```csharp
[MyAttribute(typeof(MyAttribute))]
```
to
```csharp
[MyAttribute(typeof (MyAttribute))]
```

Reply:
That's the input string. I changed it to highlight that the separator is not needed and thus removed.